Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -161,8 +161,7 @@ final class PledgeCTAContainerView: UIView {
       self.activityIndicator.centerXAnchor.constraint(equalTo: self.layoutMarginsGuide.centerXAnchor),
       self.activityIndicator.centerYAnchor.constraint(equalTo: self.layoutMarginsGuide.centerYAnchor),
       self.pledgeCTAButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minHeight),
-      self.pledgeCTAButton.widthAnchor.constraint(greaterThanOrEqualToConstant: Layout.Button.minWidth),
-      self.rootStackView.topAnchor.constraint(equalTo: self.layoutMarginsGuide.topAnchor)
```

Comment:
This constraint was causing warnings because it conflicts with the constraints applied here:

https://github.com/kickstarter/ios-oss/blob/master/Kickstarter-iOS/Views/PledgeCTAContainerView.swift#L145